### PR TITLE
Introduce devcontainer to get development environment more smoothly

### PR DESCRIPTION
**Pull request by syucream**
Try [devcontainer](https://code.visualstudio.com/docs/devcontainers/containers) as an alternative solution for https://github.com/dmm-com/airone/pull/935

### CODE REVIEW

**Comment by syucream** [comment]
Hmm it mostly works but the frontend is still something wrong on runtime ...
I guess its happend by clsx version required by notistack but I have no more information ...

image

**Comment by syucream** [comment]
After some tries, I become not to be able to reproduce https://github.com/dmm-com/airone/pull/1146#issuecomment-2094062755 ... maybe https://github.com/dmm-com/airone/pull/1146/commits/6b0d4f859f278cd3cae25dfa3d3d31cc55414617 resolve it. ( Give me feedback if you catch the same issue! )

Anyway I think its ready to use devcontainer to build development environment more smoothly! 🚀

File: .devcontainer/frontend/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "frontend",
```
Comment (syucream) [inline .devcontainer/frontend/devcontainer.json]: Hmm I guess its better to aggregate frontend/backend containers to one container.

Comment (hinashi) [reply .devcontainer/frontend/devcontainer.json]: Certainly, a single container is simpler and better.
I'll try using an Ubuntu image!

Comment (syucream) [reply .devcontainer/frontend/devcontainer.json]: cool!

**Review by hinashi — APPROVED** [review]
LGTM!

**Pull request merged**